security/tpm/test/_tpm_test.js

```diff
@@ -8,22 +8,27 @@ var tss = require("tss.js");
 var sinon = require('sinon');
 var assert = require('chai').assert;
 
-var fakeSimpleTpmClient = {
-  allowErrors: () => {return this},
-  connect: (callback) => {callback();},
-  ReadPublic: (handle, callback) => {
-    callback(null,null);
-  },
-  fakeSimpleTpmClient: () => {},
-  getLastResponseCode: () => {
-    return tss.TPM_RC.SUCCESS;
+var fakeEkKey = {
+  outPublic: {
+    asTpm2B: () => Buffer('fakeEndorsement')
   }
+};
+var fakeSrkKey = {
+  outPublic: {
+    asTpm2B: () => Buffer('fakeStorageRoot')
+  }
+};
+var fakeIdKey = {
+  outPublic: {
+    asTpm2B: () => Buffer('fakeIdentity')
+  }
+};
+var fakeTpm = new tss.Tpm();
```

Comment:
doesn't look very fake anymore now does it?

Reply:
Fixed.

---
In reply to: [166799310](https://github.com/Azure/azure-iot-sdk-node/pull/225#discussion_r166799310) [](ancestors = 166799310)